ui/app/components/secret-engine/configuration-details.hbs

```diff
@@ -5,7 +5,7 @@
 
 {{#if @configModels.length}}
   {{#each @configModels as |configModel|}}
```

Comment:
Mostly an FYI but this could be removed and we can leverage the `{{else}}` that is available when an array is empty ([docs](https://guides.emberjs.com/release/components/looping-through-lists/#toc_empty-lists))

So instead of wrapping in an `{{#if}}` that conditional can be removed. So this:

```hbs
{{#if @configModels.length}}
  {{#each @configModels as |configModel|}}
  ...
  {{/each}}
{{/if}}
```

Can instead be this!
```hbs
{{#each @configModels as |configModel|}}
  ...iterating over configModel...
 {{else}}
...logic for when the array is empty (I think it's an empty state?) 
{{/each}}
```

Reply:
TIL — thank you!